src/plugin/loader.ts

```diff
@@ -30,7 +30,7 @@ export default function loader(rawCode) {
   // Skip rest of files that are not inside /pages
   if (!this.resourcePath.startsWith(pagesPath)) return rawCode
 
-  const page = this.resourcePath.replace(pagesPath, '/', '')
+  const page = this.resourcePath.replace(pagesPath, '/', '').replace(/\\/g, '/')
```

Comment:
Nice @vimutti77 good catch!! 🔥 

Here I've did this mistake:

```suggestion
  const page = this.resourcePath.replace(pagesPath, '/').replace(/\\/g, '/')
```

there are only 2 parameters in the `.replace` so this extra parameter doesn't do anything... If you want to take advantage to remove it, perfectly, if not I will remove it.